rara-distill/litellm/Dockerfile

```
@@ -1,14 +1,16 @@
 FROM --platform=linux/amd64 ghcr.io/berriai/litellm:v1.90.0
 
 USER root
-# Prisma Python downloads a glibc Node.js binary at runtime (store_model_in_db),
-# which fails on Wolfi musl. Pre-create the nodeenv pointing at the system
-# musl-compatible Node.js so no download happens at container start.
-RUN /app/.venv/bin/python -m nodeenv --node=system \
-      /home/nonroot/.cache/prisma-python/nodeenv 2>/dev/null || true && \
-    chown -R nonroot:nonroot /home/nonroot/.cache 2>/dev/null || true
+# Wolfi runtime image has nodejs but not npm; Prisma Python (store_model_in_db)
+# needs npm to install the Prisma CLI. Install it so the system nodeenv has npm.
+RUN apk add --no-cache npm
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)